docs/src/Video.doc.js

```diff
@@ -68,6 +68,12 @@ card(
         description: 'The URL of the captions track for the video (.vtt file)',
         required: true,
       },
+      {
+        name: 'children',
+        type: 'React.Node',
+        description: `This \`children\` prop is not same as children inside the native html \`video\` element.
+          It will add overlays on top of the html video element like overlays, while under controls if they are shown.`,
```

Comment:
small nit: "It will add overlays on top of the html video element like overlays, while under controls if they are shown." doesnt sound right, maybe "Instead it serves to add overlays on top of the html video element, while still being under the video controls."

Reply:
thanks! Will update. Please always correct my english writing!